service/controller/app/v1/resource/chartoperator/resource.go

```diff
@@ -0,0 +1,153 @@
+package chartoperator
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/afero"
+	"gopkg.in/yaml.v2"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/helm/pkg/helm"
+
+	"github.com/giantswarm/app-operator/service/controller/app/v1/key"
+)
+
+const (
+	Name = "chartoperatorv1"
+
+	defaultClusterDNSIP    = "172.31.0.10"
+	chartOperatorNamespace = "giantswarm"
+	chartOperatorRelease   = "chart-operator"
+	chartOperatorVersion   = "0.9.0"
```

Comment:
This version should come from the app CR we have created for chart-operator in cluster-operator. We can then remove this hardcoding.